<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaConsensusModeWidget</class>
 <widget class="QWidget" name="MaConsensusModeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>299</width>
    <height>131</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="consensusTypeLabel">
     <property name="text">
      <string>Consensus type:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="consensusType"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="thresholdResetLayout">
     <item>
      <widget class="QLabel" name="thresholdLabel">
       <property name="text">
        <string>Threshold:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="thresholdResetButton">
       <property name="toolTip">
        <string>Reset to default value</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../../U2Gui/U2Gui.qrc">
         <normaloff>:/core/images/arrow_rotate_clockwise.png</normaloff>:/core/images/arrow_rotate_clockwise.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="thresholdLayout">
     <item>
      <widget class="QSlider" name="thresholdSlider">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="thresholdSpinBox">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="accelerated">
        <bool>true</bool>
       </property>
       <property name="suffix">
        <string>%</string>
       </property>
       <property name="prefix">
        <string/>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../U2Gui/U2Gui.qrc"/>
 </resources>
 <connections/>
</ui>
